packages/snap/src/index.ts

```diff
@@ -133,29 +149,62 @@ async function sendTransaction(tx: CeloTransactionRequest, wallet: CeloWallet) {
   return txResponse.wait()
 }
 
-async function getBIP44Node(index: number = 0): Promise<BIP44Node> {
-  // Get the bip44 node corresponding to the path m/44'/52752'/0'/0
-  const bip44Node = await snap.request({
+async function getKeyPair(snap: SnapsGlobalObject, address?: string, addressIndex: number = 0): Promise<KeyPair> {
+  const derivationPath = "m/44'/60'/0'/0"; //Todo - read from state config
```

Comment:
should this be the CELO derivation path? => m/44'/52752'/0'/0

Reply:
no it must be the same as what metamask derives

Reply:
otherwise the dapp you are interacting with will show you are connected with one address and than the one the snap tries to send with